build.sbt

```diff
@@ -9,6 +9,7 @@ lazy val root = project.
   settings(commonSettings).
   settings(unpublished).
   settings(
+    crossScalaVersions := Nil,
```

Comment:
Interesting, what's the effect of this?

Reply:
According to Eugene Yokota:
>the subproject should declare whatever the Scala versions it supports. if you have anything on root it will call `compile` on all the subproject that it aggregates, and results to the error you saw.

Reply:
Cool, thanks. This LGTM but let's wait for the github webhook incident to pass so we have CI verification on this before merging.